compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -117,7 +117,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             Intrinsic(box ref intrinsic) => self.emulate_nondiverging_intrinsic(intrinsic)?,
 
             // Statements we do not track.
-            AscribeUserType(..) => {}
+            PlaceMention(..) | AscribeUserType(..) => {}
```

Comment:
IMO this should have semantics in Miri... but I guess that is for later?

Reply:
I know way too little about miri to suggest semantics here.  There is room for later here.  Do you have an idea of what this should be?

Reply:
Something like
```rust
let _op = self.eval_place_to_op(place)?;
```

Reply:
The documentation in `mir/syntax.rs` would need to be changed to reflect this and it would need to be preserved throughout all mir dialects. So yeah, should probably be left for later